libraries/chain/include/graphene/chain/protocol/custom_authority.hpp

```diff
@@ -103,15 +115,28 @@ FC_REFLECT( graphene::chain::custom_authority_delete_operation::fee_parameters_t
 FC_REFLECT( graphene::chain::custom_authority_create_operation,
             (fee)
             (account)
-            (custom_id)
             (enabled)
             (valid_from)
             (valid_to)
             (operation_type)
-            (auth)
             (restrictions)
             (extensions)
           )
 
-FC_REFLECT( graphene::chain::custom_authority_update_operation, (fee)(account) )
-FC_REFLECT( graphene::chain::custom_authority_delete_operation, (fee)(account) )
+FC_REFLECT( graphene::chain::custom_authority_update_operation,
+            (custom_authority_to_update)
+            (fee)
+            (account)
+            (enabled)
+            (valid_from)
+            (valid_to)
+            (operation_type)
+            (restrictions)
+            (extensions)
+           )
+
+FC_REFLECT( graphene::chain::custom_authority_delete_operation,
+            (custom_authority_to_update)
```

Comment:
Usually, `fee` is first in list (affects serialization), please modify accordingly

Reply:
done